### ISSUE TITLE
Using search stops the container

### ISSUE THREAD

**Issue by m-kostrzewa** [wernight/docker-mopidy#7]
Hello,

every time I use the search feature of the web client, docker container stops.

Here's how I run mopidy:
```
docker run -d -e PULSE_SERVER=tcp:192.168.0.5:4713 -v $(pwd)/m
edia:/var/lib/mopidy/media:ro -v $(pwd)/local:/var/lib/mopidy/local -p 6600:6600 -p 6680:6680 
wernight/mopidy mopidy -v 
```

Tail of docker logs:
```
...
DEBUG    2017-02-19 18:21:46,597 [1:HttpServer] mopidy.http.handlers
  Received WebSocket message from 192.168.0.17: u'{"method":"core.library.search","params":{"any":["asdf"]},"jsonrpc":"2.0","id":56}'
INFO     2017-02-19 18:21:46,605 [1:YouTubeBackend-3] mopidy_youtube
  Searching YouTube for query 'asdf'
```

So it seems like youtube plugin is the issue.

Adding ``` -o youtube/enabled=false``` fixes the issue, but I can't search for youtube videos:

```
docker run -d -e PULSE_SERVER=tcp:192.168.0.5:4713 -v $(pwd)/m
edia:/var/lib/mopidy/media:ro -v $(pwd)/local:/var/lib/mopidy/local -p 6600:6600 -p 6680:6680 
wernight/mopidy mopidy -v -o youtube/enabled=false
```

Looking through mopidy-youtube github issues (https://github.com/mopidy/mopidy-youtube/issues) there are many issues with its backed (pafy vs youtube-dl), but there are no reported crashes.

However, I confirmed that I can use pafy directly from inside the container:
```
# docker exec -it 4e24 /bin/sh
$ python
Python 2.7.9 (default, Jun 29 2016, 13:08:31) 
[GCC 4.9.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import pafy
>>> url = "https://www.youtube.com/watch?v=5Sb8I_hZPPc"
>>> video = pafy.new(url)
>>> video.title
u'Counterphase - Shapeless Ocean'
```

Here's the output of mopidy of a fresh container:
```
libudev: udev_has_devtmpfs: name_to_handle_at on /dev: Function not implemented
INFO     2017-02-19 18:26:36,504 [1:MainThread] mopidy.__main__
  Starting Mopidy 2.1.0
DEBUG    2017-02-19 18:26:36,600 [1:MainThread] mopidy.ext
  Loading entry point: youtube = mopidy_youtube:Extension
DEBUG    2017-02-19 18:26:36,601 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-YouTube 2.0.2
DEBUG    2017-02-19 18:26:36,603 [1:MainThread] mopidy.ext
  Loading entry point: moped = mopidy_moped:MopedExtension
DEBUG    2017-02-19 18:26:36,604 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-Moped 0.7.0
DEBUG    2017-02-19 18:26:36,606 [1:MainThread] mopidy.ext
  Loading entry point: gmusic = mopidy_gmusic:GMusicExtension
DEBUG    2017-02-19 18:26:36,607 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-GMusic 2.0.0
DEBUG    2017-02-19 18:26:36,611 [1:MainThread] mopidy.ext
  Loading entry point: mpd = mopidy.mpd:Extension
DEBUG    2017-02-19 18:26:36,612 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-MPD 2.1.0
DEBUG    2017-02-19 18:26:36,612 [1:MainThread] mopidy.ext
  Loading entry point: http = mopidy.http:Extension
DEBUG    2017-02-19 18:26:36,612 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-HTTP 2.1.0
DEBUG    2017-02-19 18:26:36,613 [1:MainThread] mopidy.ext
  Loading entry point: stream = mopidy.stream:Extension
DEBUG    2017-02-19 18:26:36,613 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-Stream 2.1.0
DEBUG    2017-02-19 18:26:36,613 [1:MainThread] mopidy.ext
  Loading entry point: m3u = mopidy.m3u:Extension
DEBUG    2017-02-19 18:26:36,614 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-M3U 2.1.0
DEBUG    2017-02-19 18:26:36,614 [1:MainThread] mopidy.ext
  Loading entry point: softwaremixer = mopidy.softwaremixer:Extension
DEBUG    2017-02-19 18:26:36,614 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-SoftwareMixer 2.1.0
DEBUG    2017-02-19 18:26:36,614 [1:MainThread] mopidy.ext
  Loading entry point: file = mopidy.file:Extension
DEBUG    2017-02-19 18:26:36,615 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-File 2.1.0
DEBUG    2017-02-19 18:26:36,615 [1:MainThread] mopidy.ext
  Loading entry point: local = mopidy.local:Extension
DEBUG    2017-02-19 18:26:36,619 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-Local 2.1.0
DEBUG    2017-02-19 18:26:36,621 [1:MainThread] mopidy.ext
  Loading entry point: spotify = mopidy_spotify:Extension
DEBUG    2017-02-19 18:26:36,622 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-Spotify 3.0.0
DEBUG    2017-02-19 18:26:36,624 [1:MainThread] mopidy.ext
  Loading entry point: soundcloud = mopidy_soundcloud:SoundCloudExtension
DEBUG    2017-02-19 18:26:36,625 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-SoundCloud 2.0.2
DEBUG    2017-02-19 18:26:36,631 [1:MainThread] mopidy.ext
  Discovered extensions: youtube, moped, gmusic, mpd, http, stream, m3u, softwaremixer, file, local, spotify, soundcloud
DEBUG    2017-02-19 18:26:36,634 [1:MainThread] mopidy.config.keyring
  Fetching passwords from your keyring failed. Any passwords stored in the keyring will not be available. (dbus not installed)
INFO     2017-02-19 18:26:36,634 [1:MainThread] mopidy.config
  Loading config from builtin defaults
DEBUG    2017-02-19 18:26:36,639 [1:MainThread] mopidy.config
  Loading config from /etc/xdg/mopidy/mopidy.conf failed; it does not exist
INFO     2017-02-19 18:26:36,640 [1:MainThread] mopidy.config
  Loading config from /var/lib/mopidy/.config/mopidy/mopidy.conf
INFO     2017-02-19 18:26:36,642 [1:MainThread] mopidy.config
  Loading config from command line options
INFO     2017-02-19 18:26:36,645 [1:MainThread] mopidy.internal.path
  Creating dir /var/lib/mopidy/.cache/mopidy
INFO     2017-02-19 18:26:36,645 [1:MainThread] mopidy.internal.path
  Creating dir /var/lib/mopidy/.local/share/mopidy
DEBUG    2017-02-19 18:26:36,650 [1:MainThread] mopidy.ext
  Validating extension: youtube
DEBUG    2017-02-19 18:26:36,688 [1:MainThread] mopidy.ext
  Validating extension: moped
DEBUG    2017-02-19 18:26:36,695 [1:MainThread] mopidy.ext
  Validating extension: gmusic
DEBUG    2017-02-19 18:26:36,843 [1:MainThread] mopidy.ext
  Validating extension: mpd
DEBUG    2017-02-19 18:26:36,844 [1:MainThread] mopidy.ext
  Validating extension: http
DEBUG    2017-02-19 18:26:36,872 [1:MainThread] mopidy.ext
  Validating extension: stream
DEBUG    2017-02-19 18:26:36,873 [1:MainThread] mopidy.ext
  Validating extension: m3u
DEBUG    2017-02-19 18:26:36,874 [1:MainThread] mopidy.ext
  Validating extension: softwaremixer
DEBUG    2017-02-19 18:26:36,876 [1:MainThread] mopidy.ext
  Validating extension: file
DEBUG    2017-02-19 18:26:36,877 [1:MainThread] mopidy.ext
  Validating extension: local
DEBUG    2017-02-19 18:26:36,878 [1:MainThread] mopidy.ext
  Validating extension: spotify
DEBUG    2017-02-19 18:26:36,890 [1:MainThread] mopidy.ext
  Validating extension: soundcloud
INFO     2017-02-19 18:26:36,897 [1:MainThread] mopidy.__main__
  Enabled extensions: mpd, http, file, stream, m3u, youtube, softwaremixer, moped, local
INFO     2017-02-19 18:26:36,897 [1:MainThread] mopidy.__main__
  Disabled extensions: spotify, gmusic, soundcloud
WARNING  2017-02-19 18:26:36,897 [1:MainThread] mopidy.__main__
  Found gmusic configuration errors, the extension has been automatically disabled:
WARNING  2017-02-19 18:26:36,897 [1:MainThread] mopidy.__main__
    gmusic/username must be set.
WARNING  2017-02-19 18:26:36,897 [1:MainThread] mopidy.__main__
    gmusic/password must be set.
WARNING  2017-02-19 18:26:36,898 [1:MainThread] mopidy.__main__
  Found soundcloud configuration errors, the extension has been automatically disabled:
WARNING  2017-02-19 18:26:36,898 [1:MainThread] mopidy.__main__
    soundcloud/auth_token must be set.
WARNING  2017-02-19 18:26:36,898 [1:MainThread] mopidy.__main__
  Found spotify configuration errors, the extension has been automatically disabled:
WARNING  2017-02-19 18:26:36,898 [1:MainThread] mopidy.__main__
    spotify/username must be set.
WARNING  2017-02-19 18:26:36,898 [1:MainThread] mopidy.__main__
    spotify/password must be set.
WARNING  2017-02-19 18:26:36,899 [1:MainThread] mopidy.__main__
  Please fix the extension configuration errors or disable the extensions to silence these messages.
DEBUG    2017-02-19 18:26:38,980 [1:MainThread] mopidy.commands
  Available Mopidy mixers: SoftwareMixer
INFO     2017-02-19 18:26:38,981 [1:MainThread] mopidy.commands
  Starting Mopidy mixer: SoftwareMixer
DEBUG    2017-02-19 18:26:38,983 [1:MainThread] mopidy.commands
  Mixer volume left unchanged
INFO     2017-02-19 18:26:38,984 [1:MainThread] mopidy.commands
  Starting Mopidy audio
INFO     2017-02-19 18:26:38,986 [1:MainThread] mopidy.commands
  Starting Mopidy backends: YouTubeBackend, StreamBackend, M3UBackend, FileBackend, LocalBackend
DEBUG    2017-02-19 18:26:39,010 [1:MainThread] mopidy.file.library
  Failed expanding path ($XDG_MUSIC_DIR) from file/media_dirs config value.
INFO     2017-02-19 18:26:39,015 [1:Audio-2] mopidy.audio.actor
  Audio output set to "autoaudiosink"
INFO     2017-02-19 18:26:39,016 [1:MainThread] mopidy.internal.path
  Creating dir /var/lib/mopidy/.local/share/mopidy/local
DEBUG    2017-02-19 18:26:39,017 [1:MainThread] mopidy.local.actor
  Using json as the local library
DEBUG    2017-02-19 18:26:39,017 [1:MainThread] mopidy.local.json
  Loading library: /var/lib/mopidy/.local/share/mopidy/local/library.json.gz
INFO     2017-02-19 18:26:39,017 [1:MainThread] mopidy.local.json
  No local library metadata cache found at /var/lib/mopidy/.local/share/mopidy/local/library.json.gz. Please run `mopidy local scan` to index your local music library. If you do not have a local music collection, you can disable the local backend to hide this message.
INFO     2017-02-19 18:26:39,019 [1:MainThread] mopidy.local.library
  Loaded 0 local tracks using json
INFO     2017-02-19 18:26:39,028 [1:MainThread] mopidy.commands
  Starting Mopidy core
INFO     2017-02-19 18:26:39,057 [1:MainThread] mopidy.commands
  Starting Mopidy frontends: MpdFrontend, HttpFrontend
INFO     2017-02-19 18:26:39,060 [1:MainThread] mopidy.mpd.actor
  MPD server running at [::ffff:0.0.0.0]:6600
DEBUG    2017-02-19 18:26:39,061 [1:MpdFrontend-9] mopidy.zeroconf
  Zeroconf service "None" (_mpd._tcp at []:6600): dbus not installed; publish failed.
DEBUG    2017-02-19 18:26:39,062 [1:MainThread] mopidy.http.actor
  Starting HTTP server
INFO     2017-02-19 18:26:39,064 [1:HttpFrontend-11] mopidy.http.actor
  HTTP server running at [::ffff:0.0.0.0]:6680
INFO     2017-02-19 18:26:39,064 [1:MainThread] mopidy.commands
  Starting GLib mainloop
DEBUG    2017-02-19 18:26:39,065 [1:HttpFrontend-11] mopidy.zeroconf
  Zeroconf service "None" (_http._tcp at []:6680): dbus not installed; publish failed.
DEBUG    2017-02-19 18:26:39,066 [1:HttpFrontend-11] mopidy.zeroconf
  Zeroconf service "None" (_mopidy-http._tcp at []:6680): dbus not installed; publish failed.
DEBUG    2017-02-19 18:26:39,065 [1:HttpServer] mopidy.http.actor
  Loaded HTTP extension: mopidy
DEBUG    2017-02-19 18:26:39,066 [1:HttpServer] mopidy.http.actor
  Loaded static HTTP extension: moped
DEBUG    2017-02-19 18:26:39,066 [1:HttpServer] mopidy.http.actor
  HTTP routes from extensions: 
    u'/mopidy': <class 'mopidy.http.handlers.AddSlashHandler'>
    u'/mopidy/ws/?': <class 'mopidy.http.handlers.WebSocketHandler'>
    u'/mopidy/rpc': <class 'mopidy.http.handlers.JsonRpcHandler'>
    u'/mopidy/(.+)': <class 'mopidy.http.handlers.StaticFileHandler'>
    u'/mopidy/': <class 'mopidy.http.handlers.ClientListHandler'>
    u'/moped': <class 'mopidy.http.handlers.AddSlashHandler'>
    u'/moped/(.*)': <class 'mopidy.http.handlers.StaticFileHandler'>
    u'/': <class 'tornado.web.RedirectHandler'>
```

I tried running mopidy with ```--save-debug-log``` but it immediately breaks:
```
# docker run -d -e PULSE_SERVER=tcp:192.168.0.5:4713 -
v $(pwd)/media:/var/lib/mopidy/media:ro -v $(pwd)/local:/var/lib/mopidy/local -p 6600:6600 -p 
6680:6680 mopidy2 mopidy -v --save-debug-log
8d7ad117adb29badbdb2d68929b008a7edc4d3c57b9f30571bf87546d5ef6f27

# docker logs 8d7a
libudev: udev_has_devtmpfs: name_to_handle_at on /dev: Function not implemented
ERROR    2017-02-19 18:28:35,560 [1:MainThread] mopidy.__main__
  [Errno 13] Permission denied: '/mopidy.log'
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mopidy/__main__.py", line 68, in main
    log.setup_logging(config, verbosity_level, args.save_debug_log)
  File "/usr/lib/python2.7/dist-packages/mopidy/internal/log.py", line 68, in setup_logging
    setup_debug_logging_to_file(config)
  File "/usr/lib/python2.7/dist-packages/mopidy/internal/log.py", line 104, in setup_debug_logging_to_file
    config['logging']['debug_file'], maxBytes=10485760, backupCount=3)
  File "/usr/lib/python2.7/logging/handlers.py", line 117, in __init__
    BaseRotatingHandler.__init__(self, filename, mode, encoding, delay)
  File "/usr/lib/python2.7/logging/handlers.py", line 64, in __init__
    logging.FileHandler.__init__(self, filename, mode, encoding, delay)
  File "/usr/lib/python2.7/logging/__init__.py", line 911, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 936, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 13] Permission denied: '/mopidy.log'
Traceback (most recent call last):
  File "/usr/bin/mopidy", line 9, in <module>
    load_entry_point('Mopidy==2.1.0', 'console_scripts', 'mopidy')()
  File "/usr/lib/python2.7/dist-packages/mopidy/__main__.py", line 68, in main
    log.setup_logging(config, verbosity_level, args.save_debug_log)
  File "/usr/lib/python2.7/dist-packages/mopidy/internal/log.py", line 68, in setup_logging
    setup_debug_logging_to_file(config)
  File "/usr/lib/python2.7/dist-packages/mopidy/internal/log.py", line 104, in setup_debug_logging_to_file
    config['logging']['debug_file'], maxBytes=10485760, backupCount=3)
  File "/usr/lib/python2.7/logging/handlers.py", line 117, in __init__
    BaseRotatingHandler.__init__(self, filename, mode, encoding, delay)
  File "/usr/lib/python2.7/logging/handlers.py", line 64, in __init__
    logging.FileHandler.__init__(self, filename, mode, encoding, delay)
  File "/usr/lib/python2.7/logging/__init__.py", line 911, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 936, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 13] Permission denied: '/mopidy.log'
INFO     2017-02-19 18:28:35,417 [1:MainThread] mopidy.__main__
  Starting Mopidy 2.1.0
DEBUG    2017-02-19 18:28:35,513 [1:MainThread] mopidy.ext
  Loading entry point: youtube = mopidy_youtube:Extension
DEBUG    2017-02-19 18:28:35,515 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-YouTube 2.0.2
DEBUG    2017-02-19 18:28:35,517 [1:MainThread] mopidy.ext
  Loading entry point: moped = mopidy_moped:MopedExtension
DEBUG    2017-02-19 18:28:35,518 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-Moped 0.7.0
DEBUG    2017-02-19 18:28:35,519 [1:MainThread] mopidy.ext
  Loading entry point: gmusic = mopidy_gmusic:GMusicExtension
DEBUG    2017-02-19 18:28:35,520 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-GMusic 2.0.0
DEBUG    2017-02-19 18:28:35,525 [1:MainThread] mopidy.ext
  Loading entry point: mpd = mopidy.mpd:Extension
DEBUG    2017-02-19 18:28:35,525 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-MPD 2.1.0
DEBUG    2017-02-19 18:28:35,525 [1:MainThread] mopidy.ext
  Loading entry point: http = mopidy.http:Extension
DEBUG    2017-02-19 18:28:35,526 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-HTTP 2.1.0
DEBUG    2017-02-19 18:28:35,526 [1:MainThread] mopidy.ext
  Loading entry point: stream = mopidy.stream:Extension
DEBUG    2017-02-19 18:28:35,527 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-Stream 2.1.0
DEBUG    2017-02-19 18:28:35,527 [1:MainThread] mopidy.ext
  Loading entry point: m3u = mopidy.m3u:Extension
DEBUG    2017-02-19 18:28:35,527 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-M3U 2.1.0
DEBUG    2017-02-19 18:28:35,527 [1:MainThread] mopidy.ext
  Loading entry point: softwaremixer = mopidy.softwaremixer:Extension
DEBUG    2017-02-19 18:28:35,528 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-SoftwareMixer 2.1.0
DEBUG    2017-02-19 18:28:35,528 [1:MainThread] mopidy.ext
  Loading entry point: file = mopidy.file:Extension
DEBUG    2017-02-19 18:28:35,529 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-File 2.1.0
DEBUG    2017-02-19 18:28:35,529 [1:MainThread] mopidy.ext
  Loading entry point: local = mopidy.local:Extension
DEBUG    2017-02-19 18:28:35,533 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-Local 2.1.0
DEBUG    2017-02-19 18:28:35,535 [1:MainThread] mopidy.ext
  Loading entry point: spotify = mopidy_spotify:Extension
DEBUG    2017-02-19 18:28:35,536 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-Spotify 3.0.0
DEBUG    2017-02-19 18:28:35,537 [1:MainThread] mopidy.ext
  Loading entry point: soundcloud = mopidy_soundcloud:SoundCloudExtension
DEBUG    2017-02-19 18:28:35,539 [1:MainThread] mopidy.ext
  Loaded extension: Mopidy-SoundCloud 2.0.2
DEBUG    2017-02-19 18:28:35,545 [1:MainThread] mopidy.ext
  Discovered extensions: youtube, moped, gmusic, mpd, http, stream, m3u, softwaremixer, file, local, spotify, soundcloud
DEBUG    2017-02-19 18:28:35,547 [1:MainThread] mopidy.config.keyring
  Fetching passwords from your keyring failed. Any passwords stored in the keyring will not be available. (dbus not installed)
INFO     2017-02-19 18:28:35,548 [1:MainThread] mopidy.config
  Loading config from builtin defaults
DEBUG    2017-02-19 18:28:35,553 [1:MainThread] mopidy.config
  Loading config from /etc/xdg/mopidy/mopidy.conf failed; it does not exist
INFO     2017-02-19 18:28:35,553 [1:MainThread] mopidy.config
  Loading config from /var/lib/mopidy/.config/mopidy/mopidy.conf
INFO     2017-02-19 18:28:35,556 [1:MainThread] mopidy.config
  Loading config from command line options
INFO     2017-02-19 18:28:35,559 [1:MainThread] mopidy.internal.path
  Creating dir /var/lib/mopidy/.cache/mopidy
INFO     2017-02-19 18:28:35,559 [1:MainThread] mopidy.internal.path
  Creating dir /var/lib/mopidy/.local/share/mopidy
ERROR    2017-02-19 18:28:35,560 [1:MainThread] mopidy.__main__
  [Errno 13] Permission denied: '/mopidy.log'
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mopidy/__main__.py", line 68, in main
    log.setup_logging(config, verbosity_level, args.save_debug_log)
  File "/usr/lib/python2.7/dist-packages/mopidy/internal/log.py", line 68, in setup_logging
    setup_debug_logging_to_file(config)
  File "/usr/lib/python2.7/dist-packages/mopidy/internal/log.py", line 104, in setup_debug_logging_to_file
    config['logging']['debug_file'], maxBytes=10485760, backupCount=3)
  File "/usr/lib/python2.7/logging/handlers.py", line 117, in __init__
    BaseRotatingHandler.__init__(self, filename, mode, encoding, delay)
  File "/usr/lib/python2.7/logging/handlers.py", line 64, in __init__
    logging.FileHandler.__init__(self, filename, mode, encoding, delay)
  File "/usr/lib/python2.7/logging/__init__.py", line 911, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 936, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 13] Permission denied: '/mopidy.log'
```

I also tried playing with mopidy-youtube dependencies but to no avail:
```
FROM wernight/mopidy

USER root
RUN apt-get update
RUN apt-get install -y  gstreamer0.10-plugins-bad

RUN pip uninstall -y pafy
RUN pip uninstall -y mopidy-youtube

RUN pip install youtube-dl
RUN pip install mopidy-youtube
USER mopidy
```

**Comment by wernight** [comment]
Thanks for the report also on Mopidy. I remember it working before.

I pushed some minor changes but nothing that should affect what you got.

I tried the follow minimal steps:

 1. `$ docker run -p 6600:6600 -p 6680:6680 wernight/mopidy`
 2. http://localhost:6680/moped/#/
 3. Typed something in the search open and pressed enter, it didn't crash.

**Comment by m-kostrzewa** [comment]
Hm... how can I obtain more detailed logs?

**Comment by wernight** [comment]
`docker run ...` without `-d` can help. Also may be Mopidy has some more log files (didn't check).

**Comment by m-kostrzewa** [comment]
Could you try using the ```--save-debug-log``` mopidy option? I described that it fails in original post but was unable to solve the problem, I think it has something to do with the Dockerfile

**Comment by m-kostrzewa** [comment]
Heh, so I came back home and it worked! But why?

I was trying to run Jenkins in docker yesterday, and the JVM was complaining about not being able to reserve memory. Turns out, I had to disable PaX:

```
sysctl -w kernel.pax.softmode=1
```

Testing with this setting on and off confirms that it was the issue. Good thing I was playing with Jenkins yesterday, otherwise I would have never though about this... I believe this issue can be closed.
